nix/release/default.nix

```diff
@@ -12,34 +12,50 @@ let
   # Version from the postgrest.cabal file (gotten with callCabal2nix).
   version = postgrest.version;
 
-  # Set of files that will be published in the GitHub release.
-  releaseFiles =
-    runCommand "postgrest-release-files"
-      { inherit postgrest version; }
+  # Script for publishing a new release on GitHub.
+  github =
+    writeShellScriptBin "postgrest-release-github"
       ''
         set -euo pipefail
 
-        mkdir -p $out
-
-        tar cvJf "$out"/postgrest-v"$version"-linux-x64-static.tar.xz \
```

Comment:
I've removed the `v` prefix from `v$version` in the scripts. This means that the [changelog](https://github.com/PostgREST/postgrest/blob/master/CHANGELOG.md#701---2020-05-18) versions will now need a `v` prefix for `sed` to pick the changes and then serve as the body for the release(done with `ghr`).